components/director/internal/domain/tenant/converter.go

```diff
@@ -34,6 +41,8 @@ func (c *converter) FromEntity(in *tenant.Entity) *model.BusinessTenantMapping {
 		ID:             in.ID,
 		Name:           in.Name,
 		ExternalTenant: in.ExternalTenant,
+		Parent:         in.Parent.String,
```

Comment:
Should check if it is valid and assign "" or in.Parent.String depend on validity.

Reply:
I think that `in.Parent.String` is by default `""` if it's a `NULL` in the DB and it's a value different than `NULL` if there's a valid value for this column in the DB. Can there be a case in which `nullstring.String != ""` and `nullstring.Valid == false`?